src/sage/rings/number_field/totallyreal_data.pyx

```diff
@@ -357,11 +354,11 @@ cdef int eval_seq_as_poly_int(int *f, int n, int x):
 
         f[n]*x^n + f[n-1]*x^(n-1) + ... + f[0].
     """
-    cdef int s, xp
+    cdef int s
 
     s = f[n]
     for i from n > i >= 0:
-        s = s*x+f[i]
+        s = s * x + f[i]
```

Comment:
shouldn't this be `s*x + f[i]`?

Reply:
I do no think so. Can we please move forward ?

Reply:
https://peps.python.org/pep-0008/#other-recommendations
"If operators with different priorities are used, consider adding whitespace around the operators with the lowest priority(ies)."